### PR TITLE
Completed string-manipulations-Kasiet Rakhimdinova

### PR DESCRIPTION
**Pull request by Kasiett**
Screenshot 2023-03-23 at 8 15 41 PM

### CODE REVIEW

File: string-manipulation-1/src/is-lower-cased.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* exported isLowerCased */
+function isLowerCased(word) {
+  if (word.toLowerCase() === word) {
+    return true;
+  } else {
+    return false;
+  }
+  // eslint-disable-next-line no-unreachable
```
Comment (shawnkost) [inline string-manipulation-1/src/is-lower-cased.js:8]: Do not add any eslint rules without asking me first. If you need to add a rule to get past an error, that means you're doing something incorrect for the most part

File: string-manipulation-1/src/reverse-word.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/* exported reverseWord */
+function reverseWord(word) {
+  let str = '';
+  for (let i = word.length - 1; i >= 0; i--) {
+    str += word[i];
+  }
+  return str;
+}
+console.log(reverseWord('LearningFuze'));
```
Comment (shawnkost) [inline string-manipulation-1/src/reverse-word.js:9]: Delete any console logs. These are only used for testing / debugging purposes

**Review by shawnkost — CHANGES_REQUESTED** [review]
Nicely done Kasiet! Just a couple changes requested here

File: string-manipulation-1/src/reverse-word.js (hunk 2 above)
Comment (Kasiett) [reply string-manipulation-1/src/reverse-word.js]: Oh ok I won't , but on exercises which already pushed I think you will see some console.los as well. From now on I will keep in mind and try not to log in the file . Thank you for your very detailed oriented feedbacks , I really appreciate  it!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**